src/OrbitGl/PrimitiveAssembler.cpp

```diff
@@ -302,6 +302,14 @@ void PrimitiveAssembler::AddQuadBorder(const Quad& quad, float z, const Color& c
   AddLine(quad.vertices[3], quad.vertices[0], z, color);
 }
 
+void PrimitiveAssembler::AddAabbOutline(Vec2 pos, Vec2 size, float outline_width, float z,
```

Comment:
Why not using the `AddQuadBoarder` from above?

Reply:
For outline thickness control.

Reply:
Should we consider adjusting the other function to also add thickness control? I feel now those two kinda feel identical. At least consider renaming it and making it on-par with the existing one, such that its just another overload.

Reply:
But the function only works for an axis aligned bounding box.

Reply:
I am still not 100%. There doesn't seem to be a reason to not also support arbitrary quads here. But I think I'll just leave this to you. Wouldn't be too much of a refactor in the future.